internal/pkg/externalplugins/config.go

```diff
@@ -484,6 +486,10 @@ func validateLabelBlocker(labelBlockers []TiCommunityLabelBlocker) error {
 				return err
 			}
 
+			if len(blockLabel.Message) == 0 {
```

Comment:
I think it maybe can be empty. If it's empty we think no need to comment the message.